client/src/main/scala/com/raquo/app/todomvc/TodoMvcApp.scala

```diff
@@ -48,23 +49,39 @@ object TodoMvcApp {
 
   private val itemsVar = Var(List[TodoItem]())
   private val filterVar = Var[Filter](ShowAll)
+  private val toggleAllVar = Var[Boolean](false)
```

Comment:
The value of `toggleAllVar` is 100% derived from `itemsVar`, so it should not be an independent Var. It's like denormalizing the database – you end up having to manually maintain two Var-s when updating one of them.

For this case it's easiest to define `def allItemsCompleted(items: List[TodoItem]) = items.nonEmpty && items.forall(_.completed)`, and then use it as follows:

`allItemsCompleted(itemsVar.now())`

or

`checked <-- itemsVar.signal.map(allItemsCompleted)`

Laminar and Airstream have a few tickets open on them that in general are about being able to get .now() value from signals without needing an owner; once that lands, we could just make toggleAll a signal instead, but for now a simple function is the best.